### ISSUE TITLE
Please start wiki

### ISSUE THREAD

**Issue by IanKirwan** [ai-tor/DeepGTAV#87]
Hi @aitorzip 
Can I request that you start the wiki going and allow us to edit it. We can build up a useful 'how to' knowledge base that will hopefully help to avoid unnecesarry issues being raised.
Cheers
Ian

**Comment by xiaoli-chen** [comment]
good idea, I will contribute, I thought this is a great project for all self driving research~

**Comment by fuenwang** [comment]
agree, we can still maintain this project and make it better!
